### PR TITLE
Emerald Pouch Enhancements

### PR DESCRIPTION
**Pull request by ryanzhoudev**
Same behaviour as powders when you're pressing left control; there's also a roman numerals toggle in the settings book

EDIT: changed to the corkian amplifier system to have it always show; togglable option

### CODE REVIEW

**Comment by ryanzhoudev** [comment]
Okay didn't intend on both these commits stacking into one PR, but whatever

In the emerald pouch redir commit, I sort of reformatted the entire onTitle event, let me know if it's not what you guys want and I can try and make it more similar to the original.

Both commits have been tested in-game and do work for me.

File: src/main/java/com/wynntils/modules/utilities/events/ClientEvents.java [hunk 1]
```diff
@@ -304,14 +304,24 @@ public void chatHandler(ClientChatReceivedEvent e) {
...
-        e.setCanceled(true);
-        GameUpdateOverlay.queueMessage(McIf.getFormattedText(packet.getMessage()));
+        if (McIf.getUnformattedText(packet.getMessage()).matches("^§a\\+\\d+ §7.+§a to pouch$")) {
+            if (OverlayConfig.GameUpdate.RedirectSystemMessages.INSTANCE.redirectIngredientPouch) {
+                e.setCanceled(true);
+                GameUpdateOverlay.queueMessage(McIf.getFormattedText(packet.getMessage()));
+            }
+        }
+
+        if (McIf.getUnformattedText(packet.getMessage()).matches("^§a\\+\\d+§7.+Emeralds? §ato pouch$")) {
```
Comment (magicus) [inline src/main/java/com/wynntils/modules/utilities/events/ClientEvents.java:319]: The formatting differences seems very minor. The code would at least appear more robust if the emeralds case were checked and handled first, and so the pouch gets everything that does not match "Emeralds".

**Review by magicus — APPROVED** [review]
Overall looks good to me.

**Comment by ryanzhoudev** [comment]
Could you take a look at [https://discord.com/channels/394189072635133952/743928502314598511/894370082820595782](https://discord.com/channels/394189072635133952/743928502314598511/894370082820595782) and give me your opinions on how to handle this? If you're okay with just directly relaying the title messages into the update ticker, I can reorder the cases and unmark this as draft.

File: src/main/java/com/wynntils/modules/utilities/events/ClientEvents.java (hunk 1 above)
Comment (ryanzhoudev) [reply src/main/java/com/wynntils/modules/utilities/events/ClientEvents.java:319]: What do you mean by "the pouch gets everything that does not match 'Emeralds'"? I'm understanding you're saying to just check the emerald regex, then if it doesn't match, just assume it's ingredients?

**Comment by ryanzhoudev** [comment]
Going to close and reorg this PR.

**Pull request closed without merging**